monai/networks/nets/milmodel.py

```diff
@@ -11,6 +11,10 @@
 class MILModel(nn.Module):
     """
     Multiple Instance Learning (MIL) model, with a backbone classification model.
+    Currently, it only works for 2D images, typical use case is for classification of the
```

Comment:
looks like this model will be 2D only, the name `MILModel` is too generic and confusing cc@myron